plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/engine/copilot/CopilotCompletionEngine.java

```diff
@@ -99,7 +99,7 @@ public AIEngineResponse requestCompletion(
 
     @NotNull
     @Override
-    public void requestCompletionStream(
+    public void doRequestCompletionStream(
```

Comment:
Let's not rename base method.
We call requestCompletionStream is one method only (in AIAssistantProImpl) and we can do all changes only there. So won't change a AI engines at all.